addons/mrp/models/stock_move.py

```diff
@@ -116,9 +116,19 @@ def _qty_available(self):
     @api.multi
     @api.depends('move_lot_ids', 'move_lot_ids.quantity_done', 'quantity_done_store')
     def _qty_done_compute(self):
+        move_qty_ids = self.env['stock.move.lots'].sudo().read_group(
+            [
+                ('move_id', 'in', self.ids),
+                ('lot_id', '!=', False),
+                ('done_wo', '!=', False)
+            ],
+            ['move_id', 'quantity_done'],
+            ['move_id']
+        )
+        move_qty = {x['move_id'][0]: (x['quantity_done'], x['move_id_count']) for x in move_qty_ids}
```

Comment:
Do move_id_count isn't always equals 1 ?

The domain is not correct as it filter the lots without `done_wo` as it should only be done for the sum (not the condition)

Actually you should make 2 `read_group` to get the `lot_id` per `move_id` and the `quantity_done` `per move_id` with the domain

Reply:
done.